extension/lib/doc.js

```diff
@@ -184,10 +185,24 @@ class Doc {
             };
             if ( wordList) { total_words += wordList.length; };
         }
+
         // Update term document frequency object: 
         settings.getTermDocumentFreq(function(settingsTermDocumentFreq) {
             termDocumentFreq = settingsTermDocumentFreq;
-            self.updateTermDocumentFreq(termFreq, termDocumentFreq, settings.setTermDocumentFreq);
+            // This console.log is for monitoring the total word count as I go, to see how it progresses
+            console.log(Object.keys(termDocumentFreq).length);
```

Comment:
Purposely left this here so I can track how the dictionary size grows

Reply:
maybe use debug() in content.js? so during launch we can turn off all debugging statements in 1 line

Reply:
Good idea [thumbs up emoji]